### ISSUE TITLE
incorporate RGBD sensor in kuka_simulation

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/spartan#217]
Would be nice to have the simulator match exactly/well with the real robot. Including publishing out the pointcloud and rgb images over the appropriate channels. @kmuhlrad would you mind being in charge of this? Feel free to ping whoever you need at TRI. 

To do:

- [ ] Get RGBD information published out over the network. It would be fine to just start with depth information.
- [ ] Make URDF that (approximately) matches the iiwa_1 station in our lab.
- [ ] Connect gripper in sim to the ROS messages published by our gripper driver.

**Comment by gizatt** [comment]
fyi i have a prototype version of this on a branch. drake sim supports rgbd
publishing in lcm -- ros extension is a pretty simple substitution that i
tried.

if this is non critical happy to talk about and move on prs when i get back
on monday

On Dec 13, 2017 20:06, "Lucas Manuelli" <notifications@github.com> wrote:

Would be nice to have the simulator match exactly/well with the real robot.
Including publishing out the pointcloud and rgb images over the appropriate
channels. @kmuhlrad <https://github.com/kmuhlrad> would you mind being in
charge of this? Feel free to ping whoever you need at TRI.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/RobotLocomotion/spartan/issues/217>, or mute the thread

**Comment by manuelli** [comment]
Yeah happy to wait on this until you return.

**Comment by manuelli** [comment]
Resolved by #240
